sherpa/astro/xspec/tests/test_xspec.py

```diff
@@ -754,7 +767,7 @@ def test_evaluate_xspec_model(modelcls):
     assert_is_finite(evals, modelcls, "energy")
     assert_is_finite(wvals, modelcls, "wavelength")
 
-    assert wvals == pytest.approx(evals)
+    assert wvals == pytest.approx(evals, rel=1e-6, abs=2e-5)
```

Comment:
For XSphabs, there are now numerical difference between evaluating in energy and in wavelength space. The reason is that one of the bin boundaries (0.5 keV) is right at the boundary where I switch from one canonical value to the other and because of numerical differences energy->wavelength->energy does not end up on the same side of the 0.5 keV boundary.

I'm not happy to loosen up a general test that tests dozens of models for this one special case, but I also think it's not worth the complexity of treating just this one model different than the others.

Reply:
The wavelength checks are always less important to me ;-)

This change in tolerance doesn't appear terrible anyway.